pywinauto/unittests/test_UIAWrapper.py

```diff
@@ -60,6 +60,36 @@ def testFriendlyClassName(self):
             "Test getting the friendly classname of the dialog"
             self.assertEqual(self.button.friendly_class_name(), "Button")
 
+        def testFindNonTopCtlByClassNameAndTitle(self):
```

Comment:
It's not so critical to have all test methods PEP-8 complaint. But I think new methods might be in PEP-8. As well as doc strings.